src/world/map.c

```diff
@@ -4503,77 +4503,91 @@ void map_extend_boundary_surface()
 }
 
 /**
- * Clears all elements properly from a certain tile.
- *  rct2: 0x0068AE2A
+ * Clears the provided element properly from a certain tile, and updates
+ * the pointer (when needed) passed to this function to point to the next element.
  */
-static void clear_elements_at(int x, int y)
+void clear_element_at(int x, int y, rct_map_element **elementPtr)
```

Comment:
Ok.

In fact, the diff shows that the clear_elements_at() were replaced, when it was displaced.